templates/ai/index.html

```diff
@@ -133,16 +139,172 @@ <h2>Develop artificial intelligence projects on any environment</h2>
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/54efd5a7-tensor-flow-logo.png",
+            alt="",
+            width="371",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/5e048b8e-tensor-rt-logo.png",
+            alt="",
+            width="320",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/795f2fc2-knative-logo.png",
+            alt="",
+            width="153",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/20a0d896-pytorch-logo.png",
+            alt="",
+            width="336",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/86e6c732-seldon-logo.png",
+            alt="",
+            width="374",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/c6a757b1-pachyderm-logo.png",
+            alt="",
+            width="125",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/31c63bdf-onnx logo.png",
+            alt="",
+            width="348",
+            height="384",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/73accfaf-nvidia-triton-server-logo.png",
```

Comment:
This one looks misaligned a bit 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/4964fdb1-e99f-448c-bf49-df1a6b37139e)

Reply:
I think this is left aligned, if we align this to center than then text would be coming out on the left side.